lisa/mixin_modules.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+# The file imports all the mix-in types that can be initialized
+# using reflection.
+
+import platform
```

Comment:
What's the reason to change it like this way? It creates another place to update, and hard to troubleshoot if someone added a new module.

Reply:
Using `if platform.system() == "Linux"` checks over `import` statements is the pythonic way of handling platform specific imports. Doing it this way, allows us to use that pattern.

Reply:
The main concern is it's a static way. It's hard to troubleshoot, if someone forget to update this file.

Reply:
If I forget to add the QEMU platform then the error is: `cannot find subclass 'qemu' of runbook Platform`. That seems fairly easy to diagnose that there is something wrong with the imports.

Reply:
Ok, we can go with this way.

Reply:
Do you want me to split off this part of the change?

Reply:
Well, I prefer the previous solution, which uses a dict to deal with special modules. It doesn't need an extra line for normal modules. But this solution is ok to me too. It's not a big deal.